docs/app/pages/components/components-sections/icon/icon/icon.component.html

```diff
@@ -0,0 +1,124 @@
+    via a custom set, or an icon set created for a particular size could be used</p>
+<p>
+    The following properties can be used to configure the component:
+</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="name" type="string">
+        The name of the icon to be displayed. By default this will be one of the UX Aspects icons, unless a
+        custom icon with this name has been defined.
+    </tr>
+    <tr uxd-api-property name="size" type="string">
+        The size of the icon to be displayed. This value is in CSS units for example <code>24px</code> or
+        <code>2rem</code>.
+    </tr>
+    <tr uxd-api-property name="rotate" type="90 | 180 | 270">
+        Rotation of the icon in degrees.
+    </tr>
+    <tr uxd-api-property name="flipHorizontal" type="boolean" defaultValue="false">
+        The icon can be horizontally flipped if set to true. The default is false.”
+    </tr>
+    <tr uxd-api-property name="flipVertical" type="boolean" defaultValue="false">
+        The icon can be vertically flipped if set to true. The default is false.”
+    </tr>
+</uxd-api-properties>
+
+<p>The following code can be used to create the example above:</p>
+
+<uxd-snippet [content]="snippets.compiled.appHtml" language="html"></uxd-snippet>
+
+<h4>Overriding the Default Icon Set</h4>
+
+<p>Add this code snippet to your root module file to override icons within your whole application from a
+    different icon set. To override the icon set on a  per module basis the <code>forRoot</code> should be
+    replaced with <code>forChild</code>. This is achieved by specifying the <code>name</code> default
+    icon to be replaced, the name of the <code>icon</code> to replace it and the <code>iconset
+    </code> the icon will be from. Optionally you can add a <code>size</code> parameter in CSS units.</p>
+
+<uxd-snippet [content]="snippets.compiled.appModuleTs" language="javascript"></uxd-snippet>
+
+<p>The default icon set can be overridden by using the <code>IconService</code> on a per module basis.
+    The following functions can be called on the service:</p>
+
+<uxd-api-properties tableTitle="Icon Service">
+    <tr uxd-api-property name="setIcon" args="IconDefinition">
+        This function should be called to override a single icon from the default icon set. It should be
+        passed a <code>IconDefinition</code> object containing properties to override the default icon set.
+    </tr>
+    <tr uxd-api-property name="setIcons" args="Array<IconDefinition>">
+        This function should be called to override several icons from the default icon set in an array. It
+        should be passed an array of <code>IconDefinition</code> objects containing properties to override
+        the default icon set.
+    </tr>
+</uxd-api-properties>
+
+<p>The object to be passed to the <code>setIcon</code> and <code>setIcons</code> functions. The following
```

Comment:
Add the "IconDefinition Interface" title above this and remove `tableTitle` below.

This interface is also used in the `forRoot`/`forChild` configuration. I would summarize this as "This interface defines the properties used to create a custom icon definition."